Makefile

```
@@ -56,6 +81,69 @@ lint: ## ESLint, both stacks
+		sqlfluff lint "$$f" || rc=1; \
+	done < "$$list"; rm -f "$$raw" "$$list"; exit $$rc
+
+# Validity, deliberately NOT formatting. Prettier on these files only explodes
+# compact arrays past printWidth — churn on load-bearing files (_routes.json)
+# for no defect caught. An unparseable file is the real failure, and
+# ground-truth.json in particular has no code that reads it to fail loudly.
+#
+# The path goes in as ARGV, never interpolated into the -e source: a filename
+# containing a single quote would otherwise close the JS string literal and run
+# whatever followed, on every `make gate`.
+lint-json: ## assert every JSON file parses (tracked + untracked, minus gitignored)
```

checkmake (0.3.2)

[warning] Target body for "lint-json" exceeds allowed length of 5 lines (8).

(maxbodylength)